src/ucm/cuda/cudamem.c

```diff
@@ -40,23 +348,31 @@ static ucm_reloc_patch_t patches[] = {
     {NULL,                                          NULL}
 };
 
-ucs_status_t ucm_cudamem_install()
+static ucs_status_t ucm_cudamem_install(int events)
 {
     static int ucm_cudamem_installed = 0;
     static pthread_mutex_t install_mutex = PTHREAD_MUTEX_INITIALIZER;
     ucm_reloc_patch_t *patch;
     ucs_status_t status;
 
+    if (!(events & (UCM_EVENT_MEM_TYPE_ALLOC | UCM_EVENT_MEM_TYPE_FREE))) {
+        status = UCS_OK;
```

Comment:
minor: can init `status` to `UCS_OK` at initialization phase. Then would not need to set it to `UCS_OK` three times